tests/ParserTest.php

```diff
@@ -692,8 +692,10 @@ public function validHostProvider()
         $long_label = implode('.', array_fill(0, 62, 'a'));
 
         return [
-            'RFC3986 registered name' => ['bebe.be', true],
             'RFC3987 registered name' => ['bébé.bé', true],
+            'RFC3986 registered name (1)' => ['bebe.be', true],
+            'RFC3986 registered name (2)' => ['www._fußball.com-', true],
```

Comment:
Is `ß` in the ASCII table?

Reply:
no `ß` is not ... but then the host is considered to be in IDN form so a internal conversion is done prior to validation otherwise even `bébé.be` would fail

Reply:
Ah-ha, right. 👍